api_tests/base/test_serializers.py

```diff
@@ -15,7 +15,11 @@
 from osf_tests import factories
 from tests.utils import make_drf_request_with_version
 
+from osf.models import RegistrationSchema
+from website.project.metadata.schemas import LATEST_SCHEMA_VERSION
```

Comment:
I'd get this up to date with develop - `LATEST_SCHEMA_VERSION` is no longer a thing.  Previously, we had a bunch of different registration schemas, and the latest schema for all of them was at 2.  So LATEST_SCHEMA_VERSION was hardcoded at 2.  Once we had to move one schema to 3, this weird constant didn't work anymore and we updated logic to return whatever the latest version of each schema was, not assuming it was 2.

I would just hardcode schema version to be 2 for this test case.